cypress/component/Tag.spec.tsx

```diff
@@ -1,9 +1,73 @@
+/* eslint-disable max-nested-callbacks */
+/* eslint-disable max-lines-per-function */
 import React from 'react'
-import { Container, Tag } from '@toptal/picasso'
+import {
+  Container,
+  Settings16,
+  Tag,
+  TagProps,
+  TagRectangularProps
+} from '@toptal/picasso'
 import { TypographyOverflow } from '@toptal/picasso-lab'
+import { noop } from '@toptal/picasso/utils'
 import { mount } from '@cypress/react'
 import { TestingPicasso } from '@toptal/picasso/test-utils'
 
+interface RegularExampleArgs {
+  variant?: TagProps['variant']
+}
+
+interface CheckableExampleArgs {
+  checked?: boolean
+  hovered?: boolean
+}
+
+interface RectangularExampleArgs {
+  variant?: TagRectangularProps['variant']
+  indicator?: TagRectangularProps['indicator']
+}
+
+const renderVariantExample = ({ variant }: RegularExampleArgs) => (
```

Comment:
1. I don't see any use of `Example` postfix. I'd get rid of it.
2. Calling a type `XxxArgs` means that function `Xxx` accepts arguments of type `XxxArgs`. In your implementation, I see an inconsistency.

```suggestion
const renderRegularTags = ({ variant }: RenderRegularTagsArgs) => (
```